model/DataBaseAccess.php

```diff
@@ -177,7 +178,28 @@ public function addMultiplePermissions(array $permissionData)
                 }
             }
         }
-        $this->getPersistence()->insertMultiple(self::TABLE_PRIVILEGES_NAME, $insert);
+
+        $this->getLogger()->debug(
+            "Processing {count} permission inserts in {chunks} chunks",
+            [
+                'count' => count($insert),
+                'chunks' => ceil(count($insert)/self::INSERT_CHUNK_SIZE)
```

Comment:
Store `count($insert)` in a separate variable and use it instead.

Reply:
Changed in 0c6e230eaa4b7ffc1a0225ae00c07c1dd8942614 along with some linter errors. Thanks!